.github/workflows/docs.yml

```diff
@@ -43,3 +45,14 @@ jobs:
           publish_dir: ./target/doc/
           keep_files: false
           destination_dir: ${{ env.DEST }}/docs
+
+      - run: python3 ./gir-rustdoc/gir-rustdoc.py --project-title 'GTK Core Rust bindings' html-index
+      - name: deploy index page
+        uses: peaceiris/actions-gh-pages@v3
+        if: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/master') || github.event_name == 'release' }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./public/
+          keep_files: true
+          destination_dir: ./
+
```

Comment:
Nit: please remove this extra line.

Reply:
Done, please squash when merging :P too lazy to clone/rebase...